### PR TITLE
Update attendance til Jan 11 

### PR DESCRIPTION
**Pull request by linamnt**

### CODE REVIEW

File: Attendance.csv [hunk 1]
```diff
@@ -38,3 +38,9 @@ Date,Session,Attendance,Notes,
 2017-10-12,Coworking Git,5,2 physics 1 physiology 2 IBBME,
 2017-10-19,Intro Machine Learning,14,3 physics 1 nutritional science 1 marketing 3 molecular genetics 2 CSB 2 math/cs 1 IBBME 2 software development 1 medical biophysics,
 2017-10-26,Python scikit-learn,11,3 physics 2 IBBME 1 nutritional science 1 math 3 misc 1 physiology,
+2017-11-02,Intro NLP in Python ,8,1 physiology 2 business 1 nutritional science 2 physics 1 molecular genetics 1 psychology,
+2017-11-09,Coworking,4,2 physiology 1 neuroscience 1 physics,
+2017-11-16,Intro Bash,4,?,
```
Comment (QuLogic) [inline Attendance.csv]: 2 physics, 1 neuroscience, and Taavi?

File: Attendance.csv [hunk 2]
```diff
@@ -38,3 +38,9 @@ Date,Session,Attendance,Notes,
 2017-10-12,Coworking Git,5,2 physics 1 physiology 2 IBBME,
 2017-10-19,Intro Machine Learning,14,3 physics 1 nutritional science 1 marketing 3 molecular genetics 2 CSB 2 math/cs 1 IBBME 2 software development 1 medical biophysics,
 2017-10-26,Python scikit-learn,11,3 physics 2 IBBME 1 nutritional science 1 math 3 misc 1 physiology,
+2017-11-02,Intro NLP in Python ,8,1 physiology 2 business 1 nutritional science 2 physics 1 molecular genetics 1 psychology,
+2017-11-09,Coworking,4,2 physiology 1 neuroscience 1 physics,
+2017-11-16,Intro Bash,4,2 physics 1 neuroscience and 1 misc,
+2017-11-23,Gadgets and Hardware,13,?,
```
Comment (QuLogic) [inline Attendance.csv]: Is that including Brian?

Comment (linamnt) [reply Attendance.csv]: I believe so, but I don't have the form anymore. Do you think it's missing Brian hence there was 14 ?

Comment (QuLogic) [reply Attendance.csv]: I kinda remember everyone who was there, so yea I think it's missing Brian. I also checked the RSVPs and there are 11 + Lindsay, Madeleine, and Brian = 14.

**Pull request merged**